catalog/schematron/catalog-schematron-plugin/src/main/java/ddf/services/schematron/SchematronValidationService.java

```diff
@@ -74,537 +71,162 @@
 
             // Stage 1: Perform inclusion expansion on Schematron schema file
-            DOMResult stage1Result = performStage(bundle, schSource,
-                    ISO_SCHEMATRON_INCLUSION_EXPAND_XSL_FILENAME);
+            DOMResult stage1Result = performStage(schSource, getClass().getClassLoader().getResource("iso-schematron/iso_dsdl_include.xsl"));
             DOMSource stage1Output = new DOMSource(stage1Result.getNode());
 
             // Stage 2: Perform abstract expansion on output file from Stage 1
-            DOMResult stage2Result = performStage(bundle, stage1Output,
-                    ISO_SCHEMATRON_ABSTRACT_EXPAND_XSL_FILENAME);
+            DOMResult stage2Result = performStage(stage1Output, getClass().getClassLoader().getResource("iso-schematron/iso_abstract_expand.xsl"));
             DOMSource stage2Output = new DOMSource(stage2Result.getNode());
 
             // Stage 3: Compile the .sch rules that have been prepocessed by Stages 1 and 2 (i.e.,
             // the output of Stage 2)
-            DOMResult stage3Result = performStage(bundle, stage2Output, ISO_SVRL_XSL_FILENAME);
+            DOMResult stage3Result = performStage(stage2Output, getClass().getClassLoader().getResource("iso-schematron/iso_svrl_for_xslt2.xsl"));
             DOMSource stage3Output = new DOMSource(stage3Result.getNode());
 
-            // Precompile the Schematron preprocessor XSL file
-            this.validator = transformerFactory.newTemplates(stage3Output);
-        } catch (TransformerConfigurationException e) {
-            LOGGER.error("Couldn't create transfomer", e);
-            throw new SchematronInitializationException(
-                    "Error trying to create SchematronValidationService using sch file "
-                            + this.schematronSchemaFilename, e);
-        } catch (TransformerException e) {
-            LOGGER.error("Couldn't create transfomer", e);
-            throw new SchematronInitializationException(
-                    "Error trying to create SchematronValidationService using sch file "
-                            + this.schematronSchemaFilename, e);
-        } catch (ParserConfigurationException e) {
-            LOGGER.error("Couldn't create transfomer", e);
-            throw new SchematronInitializationException(
-                    "Error trying to create SchematronValidationService using sch file "
-                            + this.schematronSchemaFilename, e);
-
-        // Would go here if an invalid .sch file was passed in
+            template = transformerFactory.newTemplates(stage3Output);
         } catch (Exception e) {
-            LOGGER.error("Couldn't create transfomer", e);
-            throw new SchematronInitializationException(
-                    "Error trying to create SchematronValidationService using sch file "
-                            + this.schematronSchemaFilename, e);
+            LOGGER.warn("Exception compiling schematron rules ", e);
```

Comment:
Why the change from error to warn? The validator is still going to blow up if compilation fails.